src/controllers/playback/videoosd.js

```diff
@@ -1158,6 +1178,29 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
                         playbackManager.fastForward(currentPlayer);
                         showOsd();
                     }
+                //
+                case "Home":
+                    playbackManager.seekPercent(0, currentPlayer);
+                    break;
+                case "End":
+                    // Honestly this binding seems kind of pointless, just use 'nextTrack',
+                    // but YouTube has this one and that's the standard I'm aiming for
+                    playbackManager.seekPercent(100, currentPlayer);
+                    break;
+                case "0":
+                case "1":
+                case "2":
+                case "3":
+                case "4":
+                case "5":
+                case "6":
+                case "7":
+                case "8":
+                case "9":
+                    var percent = parseInt(key);
```

Comment:
The radix for parseInt does **not** default to base 10. It should be explicitly set, even if it works as is.

See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description